engine/src/main/java/org/terasology/rendering/collada/ColladaLoader.java

```diff
@@ -304,6 +309,10 @@ protected void parseSkeletalMeshData(Element rootElement) throws ColladaParseExc
             // Find a node with sid="joint-name"
             for (String jointName : jointNameArray) {
 
+                if (jointName == null)
```

Comment:
The error in LGTM is actually about `jointNameArray` being null, not `jointName`.

Reply:
fixed